pymc/distributions/continuous.py

```diff
@@ -2993,6 +2996,29 @@ def logp(value, mu, sigma, alpha):
             tau > 0,
             msg="tau > 0",
         )
+    def icdf(prob, mu, sigma, alpha, max_iter=100, tol=1e-8):
```

Comment:
What justifies the `icdf` method having a different signature compared to the `logp` method mentioned above?